samples/firefox-accounts/src/main/java/org/mozilla/samples/fxa/LoginFragment.kt

```diff
@@ -0,0 +1,87 @@
+package org.mozilla.samples.fxa
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.webkit.CookieManager
+import android.webkit.WebView
+import android.webkit.WebViewClient
+
+private const val AUTH_URL = "authUrl"
+private const val REDIRECT_URL = "redirectUrl"
+
+class LoginFragment : Fragment() {
+    private lateinit var authUrl: String
+    private lateinit var redirectUrl: String
+    private var listener: OnLoginCompleteListener? = null
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        arguments?.let {
+            authUrl = it.getString(AUTH_URL)
+            redirectUrl = it.getString(REDIRECT_URL)
+        }
+    }
+
+    @SuppressLint("SetJavaScriptEnabled")
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        val v: View = inflater.inflate(R.layout.fragment_view, container, false)
+        val wv: WebView = v.findViewById(R.id.webview)
+        // Need JS, cookies and localStorage.
+        wv.settings.domStorageEnabled = true
+        wv.settings.javaScriptEnabled = true
+        CookieManager.getInstance().setAcceptCookie(true)
```

Comment:
nit: I think `v` and `wv` are hard to read/distinguish. How about just `view` and `webView`?